contracts/modules/royalty/policies/VaultController.sol

```diff
@@ -23,14 +23,6 @@ abstract contract VaultController is IVaultController, ProtocolPausableUpgradeab
     bytes32 private constant VaultControllerStorageLocation =
         0x88cf5a7bd03e240c4fc740fb2d1a8664ec6fa4816f867d60f968080755fb1700;
 
-    /// @dev Set the snapshot interval
```

Comment:
Can remove (or DEPRECATE) `uint256 snapshotInterval;` from the variables on VaultController.sol

Reply:
yes, mark it as DEPRECATED.